apps/explorer/src/components/Layout/PageLayout.tsx

```diff
@@ -57,17 +64,22 @@ export function PageLayout({ gradient, content, loading, isError }: PageLayoutPr
 				</div>
 			)}
 			<main
-				className={clsx(
-					'relative z-10 bg-offwhite',
-					!isGradientVisible && renderNetworkDegradeBanner && 'pt-headerNetworkDegrade',
-					!isGradientVisible && !renderNetworkDegradeBanner && 'pt-header',
-				)}
+				className="relative z-10 bg-offwhite"
+				style={
+					!isGradientVisible
+						? {
+								paddingTop: `${headerHeight}px`,
```

Comment:
This is only an issue because we used a `fixed` header. any reason why we didn't use sticky, which generally avoids the need for this entirely?

Reply:
@Jordan-Mysten if we do sticky, the overlay behind the header is not going to be visible (image 1). The code previously had an overlay behind that stretching full size of the page. While this worked on the Home page because the page size is long, on other pages especially when erroring out, the overlay background will show up and it looked very wrong (image 2)

<img width="1681" alt="Screenshot 2023-08-01 at 2 45 30 PM" src="https://github.com/MystenLabs/sui/assets/127577476/1065ac83-a22a-4b8d-a8f1-99671605280d">

![image](https://github.com/MystenLabs/sui/assets/127577476/b327d1e1-0979-4012-a93e-12a17bf425b5)

Reply:
Seems solvable with negative margin?

Reply:
Although you'd _probably_ need to know the negative margin amount I suppose.

Reply:
I haven't thought about it a lot, it just seems like there's _probably_ a slightly neater way to do this but it's fine.